jupyter/trustyai/container/Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG UBI_VERSION
+ARG PYTHON_VERSION
+ARG PYTHON_VERSION_LONG
+ARG RELEASE
+ARG DATE
+ARG CUDA
+
+LABEL name="workbench-images:${CUDA}jupyter-trustyai-${UBI_VERSION}-${PYTHON_VERSION}_${RELEASE}_${DATE}" \
+    summary="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    description="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    io.k8s.description="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION} for ODH or RHODS" \
+    io.k8s.display-name="Data Science + TrustyAI with ${PYTHON_VERSION_LONG} based on ${UBI_VERSION}" \
+    authoritative-source-url="https://github.com/opendatahub-io-contrib/workbench-images" \
+    io.openshift.build.commit.ref="${RELEASE}" \
+    io.openshift.build.source-location="https://github.com/opendatahub-io-contrib/workbench-images/jupyter/trustyai" \
+    io.openshift.build.image="https://quay.io/opendatahub-contrib/workbench-images:${CUDA}jupyter-trustyai-${UBI_VERSION}-${PYTHON_VERSION}_${RELEASE}_${DATE}"
+
+USER 0
+
+RUN INSTALL_PKGS="java-11-openjdk java-11-openjdk-devel" && \
```

Comment:
Minor note, I think `java-11-openjdk` is enough, as far as I know we don't need to compile new java code (aka JDK `java-11-openjdk-devel`)

Reply:
correct, I will drop that.